src/main/java/org/schemaspy/DbDriverLoader.java

```diff
@@ -198,8 +228,8 @@ private ClassLoader getDriverClassLoader(List<URL> classpath) {
      * @return
      * @throws MalformedURLException
      */
-    private List<URL> getExistingUrls(String path) throws MalformedURLException {
-        List<URL> existingUrls = new ArrayList<URL>();
+    private Set<URL> getExistingUrls(String path) throws MalformedURLException {
+        Set<URL> existingUrls = new HashSet<>();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Use the URI class instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2112)